client/signup/steps/domains/index.jsx

```diff
@@ -186,6 +186,10 @@ export class RenderDomainsStep extends Component {
 		if ( shouldUseMultipleDomainsInCart( this.props.flowName, this.props.currentUser ) ) {
 			this.props.shoppingCartManager.addProductsToCart( [ this.props.multiDomainDefaultPlan ] );
 		}
+
+		if ( this.props.flowName === 'onboarding-pm' ) {
+			loadExperimentAssignment( 'calypso_gf_signup_onboardingpm_domains_hide_free_subdomain_v2' );
```

Comment:
How much benefit is there for pre-loading the experiment here if `ProvideExperimentData` is getting called a little bit later in the same file? Removing `includeWordPressDotCom` below may be enough in reducing SRM.

Reply:
You are right. My original thought was that `renderContent()` might contain conditions that prevent `ProvideExperimentData` from being rendered. However, if we only consider the context in `onboarding-pm` flow, it shouldn't happen. Thus, there would be no benefit at all.